lib/components/form/form_validators.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/cupertino.dart';
+
+// our lord and saviour https://regex101.com/
+final alpha = new RegExp(r'[A-Z]');
+final hashtag = new RegExp(r'^(((^| )#([a-zA-Z]*))* *)*$');
+final floatRegex = new RegExp(r'^(\d*(.|,)\d*)$');
+
+String validateTagFeald(String value) {
+  // test if a valid tag
+  if (value.contains(hashtag) || value.length == 0) {
+    return null;
+  }
+
+  // if not try to say why
+  var split = value.split(" ");
+  for (String tag in split) {
+    if (!tag.startsWith("#")) {
+      return "All tags have to start with #";
+    }
+  }
+}
```

Comment:
validateTag**Field**

og den failer pa:  #test #test #

foreslår

```suggestion
final alpha = new RegExp(r'[A-Z]');
final hashtag = new RegExp(r'^#([a-zA-Z0-9])+$');
final floatRegex = new RegExp(r'^(\d*(.|,)\d*)$');

String validateTagFeald(String value) {
  var valid = false;
  value.trim().split(" ").forEach((tag) {
    valid = tag.contains(hashtag);
  });

  return valid ? null : "Tags must start with # and only contain alphanumeric characters";
}
```